backend/tracim_backend/lib/core/userworkspace.py

```diff
@@ -153,64 +153,64 @@ def create_one(
                     user.user_id, workspace.workspace_id
                 )
             )
-        role = UserRoleInWorkspace()
-        role.user = user
-        role.workspace = workspace
-        role.role = role_level
-        role.email_notification_type = email_notification_type
+        setting = UserConfigInWorkspace()
```

Comment:
it's "config" everywhere else (and should be "user_workspace_config")
is there a reason to name it setting here?

Reply:
yes it has been changed by user_workspace_config